myapp/src/pages/LoginPage.js

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import BeatLoader from "react-spinners/ClipLoader";
+
+import InputField from "../components/InputField";
+
+function LoginPage() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const [usernameError, setUsernameError] = useState("");
+  const [passwordError, setPassswordError] = useState("");
+
+  let [loading, setLoading] = useState(false);
+
+  const navigate = useNavigate();
+
+  function setData(event) {
+    event.preventDefault();
+
+    setUsernameError("");
+    setPassswordError("");
+
+    if (username === "") {
```

Comment:
```suggestion
    if (!username) {
```
This should also be a valid argument since we are using the strict comparison (===).

Reply:
fixed it.